docs/howto_config.md

```diff
@@ -1,11 +1,8 @@
 # Configuration
 
-MicroShift can be configured in the following ways, in order of precedence:
-* Command line arguments
-* Environment variables
-* Configuration file
+MicroShift can be configured via a configuration file only.
```

Comment:
Saying "only" is relevant to someone who read the old docs, but doesn't feel necessary if we're just documenting how it works. Can we just remove this line, too, and not change the wording on what is now line 5?

Reply:
Done